api/apps/api/src/modules/clone/import/adapters/archive-reader.adapter.ts

```diff
@@ -35,22 +25,6 @@ export class ArchiveReaderAdapter implements ArchiveReader {
       new RegExp(ClonePieceRelativePaths[ClonePiece.ExportConfig].config),
     );
     if (isLeft(exportConfigOrError)) return left(archiveCorrupted);
```

Comment:
As I see it, extracting export config file logic should be placed outside `ArchiveReaderAdapter`, if we truly want to reuse `ArchiveReaderAdapter` we should return a `Readable`. Another thing that i've noticed is that the potential usage of this adapter will be mostly in geoprocessing service (in piece importers), maybe we should add it to `lib` package like `FileRepository` (port and temporary adapter).